src/main/fc/settings.c

```diff
@@ -633,6 +633,9 @@ const clivalue_t valueTable[] = {
     { "osd_cap_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 20000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, cap_alarm) },
     { "osd_alt_alarm",              VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 10000 }, PG_OSD_CONFIG, offsetof(osdConfig_t, alt_alarm) },
 
+    { "osd_ah_max_pit",             VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, 900 }, PG_OSD_CONFIG, offsetof(osdConfig_t, ahiMaxPitch) },
+    { "osd_ah_max_rol",             VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, 900 }, PG_OSD_CONFIG, offsetof(osdConfig_t, ahiMaxRoll) },
```

Comment:
Yes another naming convention for artificial horizon. We have `horizon`, `ahi`, and `ah`.

I suggest we standardise on `ah`